bin/prepare_for_multiqc.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+
+# Written by Julia Graf and released under MIT license.
+
+import pandas as pd
+import numpy as np
+from argparse import ArgumentParser
+import re
+from collections import Counter
+
+parser = ArgumentParser(description='...')
+parser.add_argument(
+    "--input",
+    required=True,
+    nargs=1,
+    help="The tsv file returned by the OPENMS_TEXTEXPORTER process."
+)
+
+parser.add_argument(
+    "--out_prefix",
+    required=True,
+    nargs=1,
+    help="The prefix for the output TSV files."
+)
+
+parser.add_argument(
+    "--mass_range",
+    required=True,
+    nargs=1,
+    help="The mhcquant pipeline parameter `digest_mass_range`."
+)
+
+parser.add_argument(
+    "--prec_charge",
+    required=True,
+    nargs=1,
+    help="The mhcquant pipeline parameter `prec_charge`."
+)
+
+parser.add_argument(
+    "--fdr_threshold",
+    type=float,
+    required=True,
+    nargs=1,
+    help="The mhcquant pipeline parameter `fdr_threshold`."
+)
+
+parser.add_argument(
+    "--quantify",
+    required=True,
+    nargs=1,
+    help="Whether quantification is enabled or not."
+)
+
+def comma_separated_list(value):
+    return value.split(',')
+
+
+parser.add_argument(
+    "--columns",
+    default=None,
+    type=comma_separated_list,
+    nargs=1,
+    help="A comma-separated list of columns to keep."
+)
+
+
+def parse_multiTSV(file_path):
```

Comment:
Add quick description of input and what the function does (parses Texexporter output in quant mode)